techniques/applications/systemUpdateCampaign/1.0/metadata.xml

```diff
@@ -0,0 +1,53 @@
+<TECHNIQUE name="System Update Campaign">
+  <DESCRIPTION>plouf</DESCRIPTION>
+  <MULTIINSTANCE>false</MULTIINSTANCE>
+  <POLICYGENERATION>separated</POLICYGENERATION>
+  <USEMETHODREPORTING>true</USEMETHODREPORTING>
+
+  <BUNDLES>
+    <NAME>rudder_system_update_campaign</NAME>
+  </BUNDLES>
+
+  <TMLS>
+    <TML name="systemUpdate"/>
+  </TMLS>
+
+  <SYSTEMVARS>
+    <NAME>AGENT_RUN_INTERVAL</NAME>
+  </SYSTEMVARS>
+
+  <SECTIONS>
+    <SECTION name="Update the system" reporting="worst-case-weighted-one" id="Update the system" component="true">
+      <SECTION name="${update}" id="system-update" multivalued="true" component="true">
```

Comment:
I am not used to this syntax in the metadata, how does it work?
Are `${}` replaced by whatever the report contains at compliance computation time?

Reply:
no idea for now, this is copy-pasted from system-update